test/conformance/chainsaw/cli/two-results-with-namespace/policy.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  annotations:
+    pod-policies.kyverno.io/autogen-controllers: none
+  name: test-jmespath
+spec:
+  admission: true
+  background: false
+  rules:
+  - match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+    name: testjmespath
```

Comment:
Please choose a different rule name, again, because it's confusing to look at.

Reply:
Done